.github/workflows/ci.yml

```diff
@@ -20,10 +28,25 @@ jobs:
           toolchain: ${{steps.current_msrv.outputs.msrv}}
           override: true
       - uses: Swatinem/rust-cache@v2.0.0
-      - run: cargo build
+      - run: cargo ${{ matrix['cargo-cmd'] }}
+
   test:
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        cargo-cmd:
+          - test --all-features
+          - test --no-default-features
     steps:
+      - name: Enable Docker Remote API on Localhost
+        shell: bash
+        run: |
+          sudo sed -i \
+            's#ExecStart=/usr/bin/dockerd#ExecStart=/usr/bin/dockerd -H tcp://127.0.0.1:2375#g' \
+            /lib/systemd/system/docker.service
```

Comment:
I guess this is good enough here. We could also consider [installing an override file](https://www.digitalocean.com/community/tutorials/how-to-use-systemctl-to-manage-systemd-services-and-units#editing-unit-files).

What do you think?

Reply:
I have no real preference here - adapted pipeline accordingly in 5921126.